src/components/likeContainer/LikeContainer.tsx

```diff
@@ -1,14 +1,33 @@
 import { useState } from 'react';
-import type { User } from '../../types';
-import LikedByContainer from '../likedByContainer/LikedByContainer';
 import PostActionBar from '../postActionBar/PostActionBar';
+import LikedByContainer from '../likedByContainer/LikedByContainer';
+import { MOCK_DATA, SAMPLES_PROFILE_PICTURE } from '../../mockText';
+import { classes } from './LikeContainer.st.css';
+import type { User } from '../../types';
 
 const LikeContainer = () => {
-    const [usersWhoLiked, setUsersWhoLiked] = useState<User[]>([]);
-    const likePost = () => {}; // unshift you to the state
+    const [usersWhoLiked, setUsersWhoLiked] = useState<User[]>([
+        { userName: MOCK_DATA.SAMPLE_NAME, profilePicture: SAMPLES_PROFILE_PICTURE[2] },
+    ]);
+
+    const likePost = (user: User) => {
+        let currentUsersWhoLiked = [...usersWhoLiked];
```

Comment:
why spread an array if theres nothing to append

whats wrong with

let currentUsersWhoLiked = usersWhoLiked
?

Reply:
because the state is immutable and I use splice for removing the user who "unliked".

https://dev.to/il3ven/common-error-accidentally-mutating-state-in-react-4ndg#:~:text=Subtler%20Way%20of%20Making%20the%20Same%20Mistake